NetKAN/ProbeControlRoom.netkan

```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version": "1",
```

Comment:
There should be no quotation marks around the 1, it's a special case due to arcane reasons long forgotten